webpack-server.js

```diff
@@ -63,7 +63,24 @@ const ourWebpackConfig = getWebpackConfig({
     tsConfigPath,
 });
 
-const mergedConfig = webpackMerge.smart(ourWebpackConfig, webpackConfigFromProject);
+// We don't want to mock anything for live server. VRT only static builds
+const vrtWebpackOverrides = {
```

Comment:
we have a separate styleguide.config for vrt in consumers - it should be there, not within the package.